docs/features/garbage_collector.md

```diff
@@ -17,6 +17,45 @@ The primary method is to use the `Terminate(context.Context)` function that is
 available when a container is created. Use `defer` to ensure that it is called
 on test completion.
 
+#### NewTerminateOptions
+
+- Since testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go/releases/tag/v0.35.0"><span class="tc-version">:material-tag: v0.35.0</span></a>
```

Comment:
Again not documented on our side, as the placeholder I'm suggesting will be replaced by the release script with the version we use (it'll be `0.35.0` but imagine we change our minds and decide to release `1.0.0`).

This is our bad, so I'm going to add it to the contributing docs asap 🙏. Please forgive me on this ping-pong.

```suggestion
- Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
```